backend/db/resolve_fetch_failures_excess_items.py

```diff
@@ -54,7 +56,8 @@ def resolve_fetch_failures_excess_items(schema=SCHEMA, use_local_db=False, cache
     # - Determine if cache OK
     for dataset in list(datasets_needed):
         path = dataset_path_map[dataset]
-        if cached_dataset_threshold_hours and was_file_modified_within_threshold(path, cached_dataset_threshold_hours):
+        if cached_dataset_threshold_hours and os.path.exists(path) and \
```

Comment:
Bug fix: If allowing use of cached dataset and the dataset had never been downloaded before (e.g. GH action), was erroring.